src/esp/metadata/MetadataMediator.h

```diff
@@ -309,14 +309,49 @@ class MetadataMediator {
     return getActiveDSAttribs()->getObjAttrFullHandle(objAttrName);
   }  // getObjAttrFullHandle
 
+  /**
+   * @brief Returns articulated object model handle in dataset corresponding to
+   * passed name as substring. Assumes articulated object model with @p
+   * artObjModelName as substring exists in this dataset.
+   * @param artObjModelName substring to handle of AO model that exists in this
+   * dataset. The actual model name will be found via substring search in the
+   * manager, so the name is expected to be sufficiently restrictive to have
+   * exactly 1 match in dataset.
+   * @return name of object attributes with handle containing @p artObjModelName
+   * or empty string if none.
+   */
+  std::string getArticulatedObjModelFullHandle(
+      const std::string& artObjModelName) {
+    attributes::SceneDatasetAttributes::ptr datasetAttr = getActiveDSAttribs();
+    // this should never happen
+    if (datasetAttr == nullptr) {
+      LOG(ERROR) << "MetadataMediator::getArticulatedObjModelFullHandle : No "
+                    "current active "
+                    "dataset specified/exists named :"
+                 << activeSceneDataset_ << ".";
+      return "";
+    }
+    return datasetAttr->getArticulatedObjModelFullHandle(artObjModelName);
+  }  // getArticulatedObjModelFullHandle
```

Comment:
This check has been obviated by other functionality since added to master.  This should be rewritten to follow the format of getObjAttrFullHandle above.

Reply:
`return getActiveDSAttribs()->getArticulatedObjModelFullHandle(artObjModelName);`?